### ISSUE TITLE
Create a list of mentioned blockchain projects in the podcast 

### ISSUE THREAD

**Issue by serejandmyself** [citizen-cosmos/Citizen-Cosmos#21]
- [ ] Create a list
- [ ] Blog post

**Comment by serejandmyself** [comment]
https://github.com/citizen-cosmos/Citizen-Cosmos/blob/master/community/A%20list%20of%20mentioned%20projects.md

**Comment by johnniecosmos** [comment]
ok so i have to listen the episode and take notes yes?

**Comment by serejandmyself** [comment]
Yes. Plus, some of the episodes have mentions of various projects in their descriptions, like so: https://www.citizencosmos.space/althea

But I wouldnt rely on this 100%

**Comment by johnniecosmos** [comment]
Gavin’s episode has been completed.

**Comment by serejandmyself** [comment]
Awesome. Feel free to carry on by episode. This will be a task for all of them

**Comment by serejandmyself** [comment]
Added episode numbers to task

**Comment by johnniecosmos** [comment]
ok so i add them on the same file yes?

**Comment by serejandmyself** [comment]
Yes. For now, lets add them all there

**Comment by johnniecosmos** [comment]
Cool will start building it up then Serj

**Comment by serejandmyself** [comment]
No rush. Its a longterm task =)

**Comment by serejandmyself** [comment]
Needs completing
